.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=true
```

Comment:
Following from the other PR, can we remove package-lock file and this change now that we have the upgrades in `package.json`?

Reply:
I'm fine either way.